src/tools/profile/read_profile.c

```diff
@@ -423,13 +423,14 @@ static int show_profile_data_accum(profile_data_t *data, options_t *opts)
     return ret;
 }
 
-KHASH_MAP_INIT_INT64(request_ids, int)
+KHASH_MAP_INIT_INT64(request_ids, size_t)
```

Comment:
uint64_t?
looks like this type is stored in the hash

Reply:
`INT64` in the name of the auxiliary macro is a type of the key
so, we can use any type with it. I just aligned it with the type of `num_records`

Reply:
the point is that the value which is stored in the hash is uint64_t